plugins/module_utils/network/nxos/config/interfaces/interfaces.py

```diff
@@ -70,12 +73,29 @@ def get_interfaces_facts(self, data=None):
 
```

Comment:
Returns 'nexus' facts by default. May want to pass 'mds' value when instantiating  Facts object.